### PR TITLE
Added functionality to remove duplicate dates

### PR DESCRIPTION
**Pull request by alexandersteele**
Now only the latest date will show if matching to a previous

### CODE REVIEW

File: src/Timeline/components/Group.js [hunk 1]
```diff
@@ -16,15 +16,26 @@ const formatTime = (timestamp) =>
...
-const mapDateAndTime = (event, idx, events) => {
+const mapDateAndTime = (event, idx, events, prevEvents) => {
   const mappedEvent = { ...event, time: formatTime(event.timestamp) };
   const isMajor = idx === events.length - 1;
+  const date = formatDate(event.timestamp, isMajor);
+
+  // Get previous date
+  const prevDate = !!prevEvents
+    ? last(prevEvents.map(mapDateAndTime))?.date
+    : null;
```
Comment (jamiemoyes) [inline src/Timeline/components/Group.js:27]: Could we replace this with 
`const prevDate = !!prevEvents
    && last(prevEvents.map(mapDateAndTime))?.date;`?

Comment (alexandersteele) [reply src/Timeline/components/Group.js:27]: Yeah sure. I might have to make the change to main after merging though as it appears I'm unable to reopen the sandbox for this particular branch...

Comment (jamiemoyes) [reply src/Timeline/components/Group.js:27]: Haha yeah no worries, will approve

Comment (jamiemoyes) [reply src/Timeline/components/Group.js:27]: Or will try approve it

**Pull request merged**